src/server/JasmineGraphInstanceService.cpp

```diff
@@ -2014,7 +2014,7 @@ void *instanceservicesession(void *dummyPt) {
                 int dataPort = stoi(workerPartition.at(2));
                 string cluster = workerPartition.at(3);
 
-                string filename = "AttrCluster_" + graphID + cluster + ".txt";
+                string filename = "attrcluster_" + graphID + cluster + ".txt";
```

Comment:
We should have some separator between the graphID and cluster. Since both of these are numbers there can be situations where these gets messed up. Also there is no digit limit between the graph ID and the cluster ID. Hence there will be confusions.